.travis.yml

```diff
@@ -116,6 +116,7 @@ cache:
   directories:
     - $HOME/locallib
     - $HOME/dojo_archive
+    - $HOME/perl5
```

Comment:
This is the perlbrew directory that our Travis image pulls in. We're not installing anything into that directory. I'm pretty sure this commit is incorrect. We install everything into `$HOME/locallib`.